keel/records/admin.py

```diff
@@ -29,8 +29,7 @@ def save_model(self, request, obj, form, change):
         check_email = User.objects.filter(email=email)
         
         if len(check_email) == 0:
-            user = User.objects.create_user(email=email)
-            user.set_password("password@123")
+            user = User.objects.create_user(email=email, password="password@123")
```

Comment:
lets not harcode the password, keep it in config instead

Reply:
fixed